<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGuiClass</class>
 <widget class="QMainWindow" name="UserGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(253, 240, 224)
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Britannic Bold</family>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select presenter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_presenters"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Britannic Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bradley Hand ITC'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Britannic Bold'; font-size:11pt;&quot;&gt;Tutorials by presenter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_tutorials_presenter"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Britannic Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current tutorial</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_title"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Presenter</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_presenter"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_minutes"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Number of likes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_likes"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Link</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_link"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Duration minutes </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_seconds"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Duration seconds</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_add">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:rgb(172, 177, 214)</string>
            </property>
            <property name="text">
             <string>Add to watch list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_next">
            <property name="font">
             <font>
              <family>Britannic Bold</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:rgb(172, 177, 214)</string>
            </property>
            <property name="text">
             <string>Go to next tutorial</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Britannic Bold</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Watch List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_watchlist"/>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Britannic Bold</family>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Choose type (CSV or HTML)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_file_type"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="font">
           <font>
            <family>Britannic Bold</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgb(172, 177, 214)</string>
          </property>
          <property name="text">
           <string>Delete tutorial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_open">
          <property name="font">
           <font>
            <family>Britannic Bold</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgb(172, 177, 214)</string>
          </property>
          <property name="text">
           <string>Open in app</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exit">
          <property name="font">
           <font>
            <family>Britannic Bold</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgb(172, 177, 214)</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
